.gitignore

```diff
@@ -16,3 +16,4 @@ examples/*.srl
 .vscode
 .gitpod.yml
 cmd/machine-controller/__debug_bin
+!pkg
```

Comment:
I left it in, since I mainly work with Ruby and `pkg/` is the default output directory for built gems (packages), so it's in my global gitignore.  Can remove it from here if it's unwanted, but it caused some initial headaches for me figuring out why git didn't want see any changes there :dizzy_face:

Reply:
Hm. I guess it's fine, I wasn't aware of this being a build directory for gems.